runtimes/haskell/lbr-plutarch/lbr-plutarch.cabal

```diff
@@ -87,7 +87,7 @@ common common-language
 library
   import:          common-language
   build-depends:
-    , base                 >=4.16
+    , base
```

Comment:
I never know whether we should ride with explicit bounds. Especially for packages that are supposed to be used by others.